src/classes/Callable_API_TEST.cls

```diff
@@ -115,4 +116,23 @@ private with sharing class Callable_API_TEST {
         Test.stopTest();
     }
 
+     /**
+     * @description Verifies if Enhanced Recurring Donations are enabled
+     */
+    @IsTest
+    private static void shouldEnableRD2WhenCrlpIsEnabled() {
```

Comment:
`shouldEnableRD2OnEnableRD2Action()` since the main cause of enablement is the Callable API request. Here we tests that wiring between Callable API and the RD2 enablement works. We do not test various conditions for which RD2 enablement delegate tests are for.